### PR TITLE
Update location-based-routing-plan.md

### PR DESCRIPTION
**Pull request by scapero**
The user experience differs from published content. This has created customer confusion and extra support tickets. The proposed change should help resolve.  The missing People modality in the given scenario was confirmed as 'by design' by Teams engineering

### CODE REVIEW

File: Teams/location-based-routing-plan.md [hunk 1]
```diff
@@ -385,7 +385,7 @@ Location-Based Routing doesn't apply to the following types of interactions. Loc
 
 A Location-Based Routing enabled user without an audio conferencing license on a PSTN call isn't allowed to start a conference with another user or PSTN number. Connecting to auto attendants or call queues is allowed.
 
-If the user has an audio conferencing license, the user must start a conference with the relevant users and call the PSTN through the conference bridge to start a conference call. If the user is already on a PSTN call, they can add another user or PSTN number to the call via escalating the call using the conferencing bridge to dial out.
+If the user has an audio conferencing license, the user must start a conference with the relevant users and call the PSTN through the conference bridge to start a conference call. If the user is already on a 1:1 PSTN call, the ‘People’ modality in the Teams client will be hidden preventing the Teams LBR user from adding a participant/escalating the existing call to a conference.  In this scenario, the call must be ended. The Teams LBR user will then need to create a conference and  using the conferencing bridge to dial out to required PSTN numbers.
```
Comment (JohanFreelancer9) [inline Teams/location-based-routing-plan.md:388]:
```suggestion
If the user has an audio conferencing license, the user must start a conference with the relevant users and call the PSTN through the conference bridge to start a conference call. If the user is already on a 1:1 PSTN call, the *People* modality in the Teams client will be hidden, preventing the Teams LBR user from adding a participant or escalating the existing call to a conference.  In this scenario, the call must be ended. The Teams LBR user will then need to create a conference and use the conferencing bridge to dial out to the required PSTN numbers. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @scapero, kindly commit the suggested changes. Many thanks.

**Comment by scapero** [comment]
Thank you Johan

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**